language/German/strings.po

```diff
@@ -925,35 +931,47 @@ msgstr "Die Bildschirmdrehung im Cocktailmodus wird nicht unterstützt.\n"
 
 #: src/frontend/mame/ui/info.cpp:135
 msgid "This system requires external artwork files.\n"
-msgstr ""
+msgstr "Dieses System erfordert externe Grafikdateien.\n"
```

Comment:
Why did you change the terminology for artwork files from “Illustrationsdateien” to “Grafikdateien”?  Remember “artwork” in MAME terms includes layouts, animation, hot areas, basic scripting, etc. and not just graphics/images.

Reply:
Simply because I I forgot about the other stuff, fixed.